package.json

```diff
@@ -5,18 +5,18 @@
   "private": true,
   "scripts": {
     "postinstall": "husky install",
-    "boot": "yarn workspaces foreach -v -pA --topological-dev --no-private run build",
+    "boot": "yarn workspaces foreach -vv -pA --topological-dev --no-private run build",
     "clean": "node scripts/clean.js",
     "storybook": "storybook dev -p 6006",
     "storybook:aksel": "yarn concurrently \"yarn watch:tw\" \"yarn storybook dev -c aksel.nav.no/website/.storybook -p 6007\"",
     "dev": "yarn workspace aksel.nav.no dev",
-    "docgen": "yarn workspaces foreach -v -p run docgen",
+    "docgen": "yarn workspaces foreach -vv -pA run docgen",
```

Comment:
Added `-A` because you now have to explicitly define which workspaces to run on. I assume `-A` was the default before.